server/config/config.go

```diff
@@ -8,12 +8,14 @@ import (
 
 const (
 	defaultMinimumFees = ""
+	defaultTxSigLimit = 7
```

Comment:
See above comment - we need to hardcode this in (otherwise a node who changed their value could break consensus)